websites/A/Animeflix/metadata.json

```
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "326694667003363328",
+		"name": "Micotsu#2907"
+	},
+	"service": "Animeflix",
+	"description": {
+		"en": "Unlimited anime without ads!"
+	},
+	"url": "animeflix.live",
```

Site supports `www` 'subdomain'.
```suggestion
	"url": ["animeflix.live", "www.animeflix.live"],
```